tools/dprint_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lineWidth": "80",
+  "indentWidth": "2",
+  "nextControlFlowPosition": "sameLine",
+  "bracePosition": "sameLine"
```

Comment:
There was a bug related to this config option that is now fixed: https://github.com/dsherret/dprint/issues/180